deploy/upstream/03_operatorsource.crd.yaml

```diff
@@ -29,6 +29,14 @@ spec:
     type: string
     description: App registry namespace
     JSONPath: .spec.registryNamespace
+  - name: DisplayName
+    type: string
+    description: Display (pretty) name to indicate the OperatorSource's name
+    JSONPath: .spec.displayName
+  - name: Publisher
+    type: string
+    description: Publisher of the OperatorSource
+    JSONPath: .spec.publisher
```

Comment:
Why are we adding these in this pr?

Reply:
I copy pasted from the manifests folder file and this showed up as a discrepancy. I am fine removing this and fixing it in a separate PR. LMK what you and the other reviewers prefer.

Reply:
No strong preference here, if it matches the manifests folder I'm fine with leaving it